configsnap

```diff
@@ -681,6 +679,8 @@ run.copy_file('/proc/mounts', fail_ok=False, sort=False)
 run.copy_file('/proc/scsi/scsi', fail_ok=False, sort=False)
 run.copy_file('/etc/sudoers', fail_ok=False, sort=False)
 run.copy_dir('/etc/sudoers.d/', fail_ok=False, sort=False)
+if os.path.exists('/etc/network/interfaces.d'):
+    run.copy_dir('/etc/network/interfaces.d', fail_ok=False, sort=False)
```

Comment:
Missing a trailing slash here, required for copy_dir to work ok
`/etc/network/interfaces.d/`